content/SCALE/SCALETutorials/Storage/ManagePoolsScale.md

```diff
@@ -103,21 +221,48 @@ Select **Confirm** then click **Start Over** to clear all changes.
 
 To save changes click **Update Pool**.
 
-### Extending a Vdev
-You cannot add more drives to an existing data VDEV but you can stripe a new VDEV of the same type to increase the overall pool size.
+### Replacing Disks to Expand a Pool
 
-To extend a pool, you must add a data VDEV of the same type as existing VDEVs.
-For example, create another mirror, then stripe the new mirror VDEV to the existing mirror VDEV.
-While on the **Devices** screen, click on the data vdev, then click **Extend**. 
+To expand a pool by replacing disks, you must individually offline each disk in the pool and replace it with a higher capacity disk following the same procedure as in [Replacing Disks]({{< relref "ReplacingDisks.md" >}}).
```

Comment:
Offlining the disk should be required only if you have to replace the disks in place.  The downside of it is reduced redundancy during the process.  Safer should be to insert the new disk into an empty enclosure slot and remove the old disk only after replace operation completed.

Reply:
```suggestion
To expand a pool by replacing disks, you must individually replace each with a higher capacity disk following the same procedure as in [Replacing Disks]({{< relref "ReplacingDisks.md" >}}).
```

Reply:
It seem the procedure also documents only the in-place replace.  We may wish to test and document other approach too.